src/get-started/create-service.md

```diff
@@ -513,9 +536,9 @@ transaction you have to create a transaction message according to the
 [uniform structure](../architecture/transactions.md#messages) developed by
```

Comment:
I'd suggest to remove "API for Transactions" section because it's not informative to the reader. Instead, it's possible to promote "API for Read Requests" to "Implement API" level.